src/index.ts

```diff
@@ -0,0 +1,36 @@
+declare interface Options {
```

Comment:
You don't need to use `declare` inside of modules, only inside of definition files (`.d.ts`)